File: Backend/src/main/java/es/fernandopal/aforoqr/api/address/Address.java
package es.fernandopal.aforoqr.api.address;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Getter
@Setter
@Entity
public class Address {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long id;
    protected String buildingNumber;
    protected String street;
    protected String city;
    protected String country;

    @Override
    public String toString() {
        return street + ", " + buildingNumber + " (" + city + ")";
    }
}

File: Android/app/src/main/java/es/fernandopal/aforoqr/data/model/Address.kt
/**
 * Copyright (c) 2022 Fernando Palomo <mail@fernandopal.es>.
 * All rights reserved.
 */
package es.fernandopal.aforoqr.data.model

class Address {
    val id: Long? = null
    val buildingNumber: String? = null
    val street: String? = null
    val city: String? = null
    val country: String? = null

    override fun toString(): String {
        return "$street, $buildingNumber ($city)"
    }
}